scripts/flow/xplat.js

```diff
@@ -7,6 +7,9 @@
  * @flow
  */
 
+// eslint-disable-next-line no-unused-vars
+declare const RN$Bridgeless: boolean;
```

Comment:
Can you put this in `react-native-host-hooks.js` to match `nativeFabricUIManager` instead? The xplat.js flow file is for internal-only modules but this global will be read in open source builds.